js/components/MilkdropWindow/index.js

```diff
@@ -125,4 +129,8 @@ class MilkdropWindow extends React.Component {
   }
 }
 
-export default MilkdropWindow;
+const mapStateToProps = state => ({
+  status: state.media.status
+});
+
+export default connect(mapStateToProps)(MilkdropWindow);
```

Comment:
Ultimately I'd like to have Butterchurn not interact with Redux directly, and only talk to the Webamp API, which will need to be expanded to support Butterchurn's use case. No worries though, that's something I can do on my own. Just getting it to work is a total win.

Reply:
oh yea, I remember talking about the plugin API.  That definitely makes sense as opposed to starting to bake things into webamps redux state.

Sounds good!